docs/faq.md

```diff
@@ -0,0 +1,252 @@
+
+Syne Tune provides a command line launcher in `benchmarking/cli/launch_hpo.py`. It allows to launch a range of experiments with a single command. The CLI is documented at https://github.com/awslabs/syne-tune/blob/main/docs/command_line.md. In order to use the CLI with your own training script, you need add some meta-information, as explained in https://github.com/awslabs/syne-tune/blob/main/docs/benchmarks.md.
+
+Another possibility is to write a custom loop over different methods and benchmarks and an example is provided 
+and documented in this [directory](benchmarking/benchmark_loop/README.md).
+
+
+
+
+### <a name="schedulers-supported"></a> What different schedulers do you support? What are the main differences between them?
+
+We refer to HPO algorithms as *schedulers*. A scheduler decides which configurations to assign to new trials, but also when to stop a running or resume a paused trial. Some schedulers delegate the first decision to a *searcher*. The most important differences between schedulers in the single-objective case are:
+
+* Does the scheduler stop trials early or pause and resume trials (`HyperbandScheduler`) or not (`FIFOScheduler`). The former requires a resource dimension (e.g., number of epochs; size of training set) and slightly more elaborate reporting (e.g., evaluation after every epoch), but can outperform the latter by a large margin.
+* Does the searcher suggest new configurations by uniform random sampling (`searcher='random'`) or by sequential model-based decision-making (`searcher='bayesopt'`, `searcher='kde'`). The latter can be more expensive if a lot of trials are run, but can also be more sample-efficient.
+
+An overview of this landscape is given in https://github.com/awslabs/syne-tune/blob/main/docs/schedulers.md. Further schedulers provided by Syne Tune include:
+
+* Population based training (PBT): https://github.com/awslabs/syne-tune/blob/main/examples/launch_pbt.py
+* Multi-objective asynchronous successive halving (MOASHA): https://github.com/awslabs/syne-tune/blob/main/examples/launch_moasha_instance_tuning.py
+* Constrained Bayesian optimization: https://github.com/awslabs/syne-tune/blob/main/examples/launch_bayesopt_constrained.py
+* Bayesian optimization by density-ratio estimation (BORE): https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/baselines.py#L75
+* Regularized evolution: https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/baselines.py#L87
+* Median stopping rule: https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/schedulers/median_stopping_rule.py
+* Synchronous Hyperband: https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/schedulers/synchronous/hyperband_impl.py#L42
+* Transfer learning schedulers: https://github.com/awslabs/syne-tune/blob/main/examples/launch_nas201_transfer_learning.py
+* Wrappers for Ray Tune schedulers: https://github.com/awslabs/syne-tune/blob/main/examples/launch_height_ray.py
+
```

Comment:
Add a line advertising the short names in baselines.py. Sorry, I forgot to do that back then.